.github/workflows/ci.yml

```diff
@@ -80,3 +81,18 @@ jobs:
         NODE_ENV: disable-logging
       run: |
         npx jest -c jest/integration.config.js "${{ matrix['test-suite'] }}" --bail  --forceExit --detectOpenHandles --runInBand
+  subscribe:
+    needs: ci
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.19'
+    - name: run subscriber script at rpc-provider (2.7.1)
```

Comment:
We should just run all the examples as part of CI.
No need to make special case for this example, if it's really a test then we should make sure this stuff is in an e2e test (not sure if there is one already)

Reply:
The reason I added it to script was, I can pin/fix the roc-provider dependency to 2.7.1.. 
For jest test, it takes from node modules which has the latest packages always..
we already have a test for subscription at - https://github.com/cennznet/api.js/blob/674f3e22fa9dd4ff5a044d8127dc0c7628eeef1a/packages/api/test/e2e/rxapi.queries.e2e.ts#L39

Reply:
it feels too hacky like this, it's not quite a sensible example e.g. uses onfinality node vs. local node and why pin dependency, doesn't make sense to someone looking at examples.
we  could create the same kind of test using a new test only directory? and have nice example here for subscriptions (example is optional :))